demonstrations/tutorial_implicit_diff_susceptibility.py

```diff
@@ -0,0 +1,656 @@
+r"""
+.. _implicit_diff_susceptibility:
+
+Implicit differentiation of variational quantum algorithms
+==========================================================
+
+.. meta::
+   :property="og:description": Implicitly differentiating the the solution of a VQA in PennyLane.
+   :property="og:image": https://pennylane.ai/qml/_images/descartes.png
+
+.. related::
+
+
+*Authors: Shahnawaz Ahmed, Juan Felipe Carrasquilla Álvarez.*
+
+— Posted: 21 Nov 2022. Last updated: 21 Nov 2022.
+*Email: shahnawaz.ahmed95@gmail.com*
+
```

Comment:
```suggestion
In this demo we show how implicit gradients can be computed using a variational
algorithm written in *PennyLane* and powered by a modular implicit differentiation
implementation provided by the tool *JAXOpt* [#Blondel2021]. We compute the generalized
susceptibility for a spin system by using a variational ansatz to obtain a
ground-state and implicitly differentiating through it. In order to compare
the implicit solution, we find the exact ground-state through eigendecomposition
and determine gradients using automatic differentiation.
Even though this calculation may become unfeasible for larger systems, for a small number of spins, our method suffices.

Introduction
------------

```
@KetpuntoG , I (edited and) transferred this paragraph here as the lede. @co9olguy might want to confirm these are the highlights.

Reply:
I'd prefer (and recommended this to @quantshah) that we start demos with an accessible hook before providing a summary

Reply:
Then I suggest to take this (edited) paragraph and put it back at the end of (what I'm now calling) the Introduction instead. :)
Would prefer a short-and-sweet lede, as well.